etcd/tests/test_etcd.py

```diff
@@ -32,25 +34,54 @@
 ]
 
 
-def test_bad_config(main_instance, aggregator):
-    instance = {'url': '{}/test'.format(main_instance['url'])}
+@preview
+def test_check(aggregator, instance):
+    check = Etcd('etcd', {}, {}, [instance])
+    check.check(instance)
+
+    for metric in itervalues(METRIC_MAP):
+        try:
+            aggregator.assert_metric(metric)
+        except AssertionError:
+            pass
+
+    assert aggregator.metrics_asserted_pct > 80
```

Comment:
Is this value arbitrary ?

Reply:
Yes, not all metrics will be present all the time